temporalio/bridge/worker.py

```diff
@@ -366,3 +366,14 @@ async def encode_completion(
                 )
                 for val in command.start_child_workflow_execution.memo.values():
                     await _encode_bridge_payload(val, codec)
+
+
+def encode_search_attributes(
+    attrs: temporalio.common.SearchAttributes,
+    payloads: Mapping[str, temporalio.bridge.proto.common.Payload],
+) -> None:
+    """Encode search attributes as bridge payloads."""
+    for k, vals in attrs.items():
+        payloads[k].CopyFrom(
```

Comment:
Does this create the key `k` if it doesn't exist?

Reply:
Or do you always make sure it exists before sending to this method?

Reply:
This is a Protobuf oddity. They lazily create everything on first scalar (i.e. non-message) update. See https://developers.google.com/protocol-buffers/docs/reference/python-generated#map-fields. If I tried to do `payloads[k] = some_message`, I'd get an error because no Python Protobuf code allows assignment to a message.